addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/Log4ShellScanRule.java

```diff
@@ -39,7 +39,9 @@
 
     private static final Logger LOGGER = LogManager.getLogger(Log4ShellScanRule.class);
     private static final String PREFIX = "ascanalpha.log4shell.";
-    private static final String[] ATTACK_PATTERNS = {
+    private static final String PREFIX_44228 = "ascanalpha.log4shell.44228.";
```

Comment:
```suggestion
    private static final String PREFIX_CVE44228 = PREFIX + "cve44228.";
```

Reply:
Please add "cve" as well.

Reply:
...oh I missed that. Fixed now.

Reply:
Could you add to the (all) constants as well?